src/include/wiredtiger.in

```diff
@@ -1897,6 +1907,25 @@ struct __wt_session {
 	int __F(timestamp_transaction)(WT_SESSION *session, const char *config);
 	/*! @} */
```

Comment:
This grouping is wrong -- the `@}` should appear after the new timestamp function.